manifests/plugin/tasks.pp

```diff
@@ -4,14 +4,41 @@
 #
 # === Parameters:
 #
-# $package:: Package name to install, use ruby193-rubygem-foreman-tasks on Foreman 1.8/1.9 on EL
+# $package::        Package name to install, use ruby193-rubygem-foreman-tasks on Foreman 1.8/1.9 on EL
+#
+# $service::        Service name
+#
+# $manage_config::  If we should manage the config file for foreman-tasks. Defaults to false
+#
+# $logger_dynflow:: Logging configuration for dynflow. Defaults to true
+#
+# $logger_action::  Logging configuration for action. Defaults to true
+#
+# $cleanup_after::  Cleaning configuration: how long should the actions be kept before deleted
+#                   by `rake foreman_tasks:clean` task. Defaults to undef
 #
 # $service:: Service name
 #
 class foreman::plugin::tasks(
-  $package = $foreman::plugin::tasks::params::package,
-  $service = $foreman::plugin::tasks::params::service,
+  $package        = $foreman::plugin::tasks::params::package,
+  $service        = $foreman::plugin::tasks::params::service,
+  $manage_config  = $foreman::plugin::tasks::params::manage_config,
+  $logger_dynflow = $foreman::plugin::tasks::params::logger_dynflow,
+  $logger_action  = $foreman::plugin::tasks::params::logger_action,
+  $cleanup_after  = $foreman::plugin::tasks::params::cleanup_after,
 ) inherits foreman::plugin::tasks::params {
+  if $manage_config {
```

Comment:
Should I open a bug for the spec file, so foreman-tasks to be renamed to foreman_tasks?